src/main/java/entity/UserFactory.java

```diff
@@ -1,9 +1,6 @@
 package entity;
 
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Great idea! Rather than importing individual use classes, such as ArrayList, Date, List, Map, HashMap, etc., calling
import java.util.*; imports all the classes & interfaces from the java.util package.